examples/v2/iam_custom_role/jinja/project_custom_role.yaml

```diff
@@ -23,5 +23,5 @@ resources:
     title: My Title
     description: My description.
     includedPermissions:
-    - bigquery.config.get
```

Comment:
Just curious why this change

Reply:
bigquery.config.get is not supported in custom roles: https://cloud.google.com/iam/docs/permissions-change-log